### ISSUE TITLE
Show the contact list entry being added to the address book in the associated vote

### ISSUE THREAD

**Issue by john-light** [AutarkLabs/open-enterprise#1523]
When I created a proposal to add "J" to my org's Address Book app, the proposal wasn't very informative about what is actually happening:

![Screenshot_2019-11-02 Aragon](https://user-images.githubusercontent.com/9424721/68078454-23e62b80-fdac-11e9-96ca-12ecc309473d.png)

As you can see it shows an address label with a label I've given to the address "John" and a string that says "metadata QmdjowKKRn13DsRLFgBXwRFAH5UgtgNf5HorUiq5o46HiP" which isn't very informative.

After approving the vote, this entry showed up in the address book:

![Screenshot_2019-11-02 Aragon(1)](https://user-images.githubusercontent.com/9424721/68078468-71fb2f00-fdac-11e9-8bbb-1bab46a4dda3.png)

I would expect the proposal text to instead say something like:

> Address Book: Add [Ethereum address] to the registry as an Individual with the name "J". 

This would be a more complete and human-friendly description of what the user is actually trying to do.

**Comment by stellarmagnet** [comment]
Totally agree John!

This requires an enhancement to radspec since we are storing the name in an IPFS file. We went with IPFS so more metadata can be associated with addresses down the line, besides type.

Once the ipfs helper is implemented in radspec, we should be able to have this level of detail. In the next weeks we will be looking into that enhancement.

You’ll see similar stuff in our projects app, that also will require the helper.

**Comment by john-light** [comment]
> In the meantime, you can browse that IPFS hash to verify what is being proposed.

@stellarmagnet  do you think this hash could be turned into a link that goes to a gateway to view the file?

**Comment by stellarmagnet** [comment]
> > In the meantime, you can browse that IPFS hash to verify what is being proposed.
> 
> @stellarmagnet do you think this hash could be turned into a link that goes to a gateway to view the file?

Yes, that is one idea. We didn’t want to hardcore our gateway that we are pinning to in the radspec though. 

So this can be another potential radspec enhancement where it generates the URL based on the IPFS gateway in your global preferences.
